TileFormats/Batched3DModel/README.md

```diff
@@ -119,16 +120,20 @@ The vertex shader can be modified at runtime to use `a_batchId` to access indivi
 
 When a Batch Table is present or the `BATCH_LENGTH` property is greater than `0`, the `batchId` attribute (with the parameter semantic `_BATCHID`) is required; otherwise, it is not.
```

Comment:
The gltf snipped above is still based on 1.0 techniques/shaders. For 2.0 all that's required is a primitive contains a `_BATCHID` attribute.